app/src/renderer/components/governance/PageGovernance.vue

```diff
@@ -7,6 +7,7 @@
     data-title="Governance"
   >
     <tm-btn
+      v-if="session.signedIn"
```

Comment:
Should this not be there when not signed in or should an action modal say "Please sign in"?

Reply:
ahh good point. the button should be present, the action modal should contain the sign in message.

Reply:
i don't know how to solve this now. i'd rather have it hidden then have the double button stack. so if it's ok with you, i would leave this as is for now.

Reply:
This would be inconsistent thoughout the app as on some routes the sign in is there and on others it isn't. Imo we should fix the header design so this "stacking" is not an issue anymore.

Reply:
i believe the sign in is present everywhere. what's unique is the placement of this create button actually - which i need to find a different spot for.

Reply:
Ah right this is the create button. Sorry.

Reply:
https://github.com/cosmos/voyager/issues/2157